src/WebJobs.Script.WebHost/Configuration/ScriptApplicationHostOptionsSetup.cs

```diff
@@ -71,6 +71,14 @@ public void Configure(string name, ScriptApplicationHostOptions options)
 
         private bool IsZipDeployment(out bool isScmRunFromPackage)
         {
+            // Flex consumption does not support mutable filesystems, and therefore always counts as zip deployment. It does not use scmRunFromPackage.
+            // TODO: revisit whether this flex consumption path should return false when in placeholder mode? Thats how it appears to work for other SKUs.
```

Comment:
Can we have an issue tracking this item?

Reply:
https://github.com/Azure/azure-functions-host/issues/9971